tendrl/alerting/objects/node_context/__init__.py

```diff
@@ -0,0 +1,70 @@
+import os
+import socket
+import uuid
+
+from tendrl.alerting import objects
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils import cmd_utils
+
+
+class NodeContext(objects.AlertingBaseObject):
```

Comment:
No its not writing to etcd this is not EtcdObj. Infact I don't have EtcdObj class corresponding to this